CMakeLists.txt

```diff
@@ -8,23 +31,14 @@ if (POLICY CMP0063)
   cmake_policy (SET CMP0063 NEW)
 endif (POLICY CMP0063)
 
-project (glog)
```

Comment:
I am 👍 on this, and checked that our [minimum CMake version](https://cmake.org/cmake/help/v3.0/command/project.html) supports this syntax in `project` (I think it must have been older 2.x versions that didn't).

Reply:
I'm not. The policies must be set before a `project` call, not after. Please revert.

Reply:
`project(VERSION)` came in CMake 3.0 (i.e. not available in 2.8.12)
Also Trusty cmake package is 2.8.12 BUT ubuntu provides 3.5.1 as `cmake3` package for trusty (and travis has trusty with 3.8 or 3.9 BTW).
So it should be reasonable to think everyone have at least CMake 3.0 with VERSION support...

Reply:
And we're using `cmake_minimum_required (VERSION 3.0)`, so it'll be a very explicit error if 2.x is used. It is not worth attempting to support CMake 2, it's possibly worse than than the Python 2 and 3 split IMHO.